proxy/src/main/java/ru/leymooo/botfilter/BotFilter.java

```
@@ -74,9 +74,13 @@ public BotFilter(boolean startup)
         Settings.IMP.reload( new File( "BotFilter", "config.yml" ) );
         Scoreboard.DISABLE_DUBLICATE = Settings.IMP.FIX_SCOREBOARD_TEAMS;
         checkForUpdates( startup );
-        if ( !CachedCaptcha.generated )
+        //Глушим исключение при попытке сгенерировать капчу. Если капча уже генерируется и бот фильтр был почему-то
+        //перезапущен, капча сгенерируется все равно и доступ к ней будет прежний.
+        try
         {
             CaptchaGeneration.generateImages();
+        } catch ( CaptchaGenerationException ignored )
```

Don't just ignore Exceptions. Either provide a config option for that or print a message every time

---

> Don't just ignore Exceptions. Either provide a config option for that or print a message every time

The main reason why the exception is ignored here is that when the user executes the /botfilter reload command, it may happen that at that moment the captcha is already in the process of being generated (for example, execute the /botfilter reload command twice), in this case it makes no sense to show the stacktrace to the user in which has no information at all.
I think this problem can be solved if you forcibly stop the generation and start it again

---

Alright